build/tests.mk

```diff
@@ -49,6 +49,39 @@ crypto-tests = \
 	${1} test/coconut_test.lua && \
 	${1} test/crypto_abc_zeta.lua
 
+cortex-m-crypto-tests = \
+	${1}test/octet.lua && cat ./stdout &&\
```

Comment:
You did not check if the file `stdout` was generated by previous command or something even earlier. That is, if LUA script was returning 0 without generating file `stdout`. The sequence would not be aware.